packages/angular/create/README.md

```diff
@@ -0,0 +1,19 @@
+# `@angular/create`
+
+# Create an Angular CLI workspace
+
+Scaffold an Angular CLI workspace without needing to install the Angular CLI globally. All of the [ng new](https://angular.io/cli/new) options and features are supported.
+
+# Usage
+
+NPM
+
+```
+npm init @angular@latest [project-name] -- [...options]
```

Comment:
Nit: Is `@latest` necessary?

Reply:
Yes, for NPM this is needed as otherwise if this command was previously used, the cached binary will be used.